pom.xml

```diff
@@ -557,6 +553,11 @@
         <artifactId>jts-core</artifactId>
         <version>1.16.1</version>
       </dependency>
+      <dependency>
+        <groupId>org.xerial.snappy</groupId>
+        <artifactId>snappy-java</artifactId>
+        <version>1.1.8.2</version>
```

Comment:
I thought that was only done if the version is needed in multiple locations. The variable above `jts-core` has the version set inline.

Reply:
Ah that is good to know. There is an open pr for dependency/pom cleanup so maybe this can be updated with those changes. Also maybe an enforcement can be added to the datawave parent poms then to keep all the applications in sync.